readme.md

```diff
@@ -133,7 +133,15 @@ Babel transpiles ES6 syntax to ES5 except for...of, spread, array destructuring,
 https://babeljs.io/docs/usage/caveats/#polyfills
 
 ### Automatic incremental builds
-`npm run -s build:watch` will monitor and automatically rebuild the JavaScript and CSS bundles when files change. When used with NPM linking, this allows the app to always pull the latest local changes. Demos are viewable by running `npm run -s dev` which also updates the browser instantly for changes. Note: these demos only run in Chrome and Chromium presently.
+`npm run -s build:watch` will monitor and automatically rebuild the JavaScript and CSS bundles when files change. When used with NPM linking, this allows the app to always pull the latest local changes.
+
+### Demo
+Demos are viewable by running `npm run -s dev` which also updates the browser instantly for changes. Note: these demos only run in Chrome and Chromium presently.
```

Comment:
I thought the demos worked on all modern browsers now? See https://github.com/wikimedia/wikimedia-page-library/pull/76#issuecomment-326356868.

Reply:
I think they work in Safari etc now, though perhaps not as speedily.

Reply:
Sorry @nambatee and @montehurd, party foul! I didn't realize until now that this was in the old readme and you were just reformatting a little D:! Thanks for this improvement! :+1: